docker/actinia-stac-tests/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM mundialis/actinia-core:2.0.0 as actinia_test
+
+LABEL authors="Carmen Tawalika,Anika Weinmann"
+LABEL maintainer="tawalika@mundialis.de,weinmann@mundialis.de"
+
+ENV SETUPTOOLS_SCM_PRETEND_VERSION=0.0.0
```

Comment:
`SETUPTOOLS_SCM_PRETEND_VERSION` is only necessary when the `.git` folder is not present where `make install` is called. To have the `.git` folder copied with GHA, the depth can be given like [here](https://github.com/mundialis/actinia_core/blob/main/.github/workflows/docker.yml#L55-L58). Then the git tag is read as version instead of 0.0.0.

This is only an enhancement suggestion as for the tests the version `0.0.0` is not really important, so not a necessary change request from my side.